### DISCUSSION TITLE
Can it be used in production?

### DISCUSSION THREAD

**Discussion by mackdunkan** [elm-time/elm-time#24] [Q&A] [answered]
I have a goods logistics application on Elixir, with approximately more than a million records in the database. Will pine cope with requests and storage?

**Comment by Viir** [comment, marked as answer]
Hi Alexander! Currently, I cannot recommend Pine for a database of this size. The work to improve efficiency is ongoing.

**Reply by mackdunkan** [reply]
Thanks for the answer!
